docroot/sites/all/modules/dev/wsuser/wsuser.install

```diff
@@ -46,6 +46,13 @@ function wsuser_update_7003() {
   }
 }
 
+/**
+ * Add user permissions for all users to cancel their own accounts.
+ */
+function wsuser_update_7004() {
```

Comment:
I had already added this to the upgrade feature, ws_d7_upgrade_features.features